app/domain/qualification/expiration.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2024, Schweizer Alpen-Club. This file is part of
+#  hitobito_sac_cas and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_sac_cas
+
+class Qualification::Expiration
```

Comment:
```suggestion
class Qualification::Expiring
```

die Klasse sollte nur die abgelaufenen Qualifications zurück geben. Mail wird direkt aus dem Job selber versendet.

def entries(finish_at)
...

welches mail dann wann gesendet wird wird im Job gehandelt

Die Idee dahinter ist das man diese Klasse dann auch für mögliche andere Use Cases benutzen kann. z.B. auflisten aller Qualis die bald ablaufen im UI.

so wäre es dann auch mehr nach dem Single Responsibility Principle